tf-modules/s3-credentials/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "s3_credentials_redirect_uri" {
+  value = "${var.tea_external_api_endpoint}redirect"
```

Comment:
This **was** using the TEA internal endpoint in the original output definition (see `tf-modules/distribution/outputs.tf`), but that doesn't seem like it makes any sense. If the user is proxying their API via cloudfront or anything, then we want to use that to build the URL for the redirect, which is reflected in the TEA external endpoint output.

Reply:
I _think_ you're right that the external one makes sense.